database/tbcd/level/upgrade.go

```diff
@@ -8,10 +8,246 @@ import (
 	"context"
 	"encoding/binary"
 	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	"github.com/mitchellh/go-homedir"
+	cp "github.com/otiai10/copy"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/util"
+	"golang.org/x/sys/unix"
 
 	"github.com/hemilabs/heminetwork/database/level"
+	"github.com/hemilabs/heminetwork/rawdb"
+)
+
+var (
+	upgradeVerbose = true
+	batchSize      = 1_000_000     // move one million records per batch
+	chunkSize      = 1_000_000_000 // 1GB
```

Comment:
Is there a reason to use GB over GiB here?
```suggestion
	chunkSize      = 1024 * 1024 * 1024 // 1 GiB
```